frontend/packages/pipelines-plugin/src/utils/pipeline-actions.tsx

```diff
@@ -149,7 +149,7 @@ const rerunPipeline: KebabAction = (
 };
 
 export const rerunPipelineAndStay: KebabAction = (kind: K8sKind, pipelineRun: PipelineRun) => {
-  return rerunPipeline(kind, pipelineRun, null, null);
+  return rerunPipeline(kind, pipelineRun, null);
```

Comment:
no need to pass  `null`
```suggestion
  return rerunPipeline(kind, pipelineRun);
```

Reply:
I think we can safely remove the third argument `resources` from the `rerunPipeline` function as it not used anywhere.

Reply:
Yeah I also thought about that and it makes sense to remove this.

Reply:
100% agree, why was this changed in the first place? @vikram-raj Based on the git history, you just added two `null` arguments onto the end of this function call during I18n work.

Reply:
Done